examples/bootstrap.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable */
+const { initialize } = require('../lib');
+
+const data = [
+  {
+    enabled: true,
+    name: 'bbbbbb',
+    description: '',
+    project: 'default',
+    stale: false,
+    type: 'experiment',
+    variants: [],
+    strategies: [],
+  },
+  {
+    enabled: true,
+    name: 'undefinedunleash-e2e-22',
+    description: 'hellowrdada',
+    project: 'default',
+    stale: false,
+    type: 'release',
+    variants: [],
+    strategies: [],
+  },
+  {
+    enabled: false,
+    name: 'you',
+    description: '',
+    project: 'default',
+    stale: false,
+    type: 'release',
+    variants: [],
+    strategies: [],
+  },
+];
+
+const client = initialize({
+  appName: 'my-application',
+  url: 'http://bogous-url:3000/proxy/',
+  customHeaders: {
+    Authorization: 'bootstrap',
+  },
+
+  bootstrap: {
+   url: 'http://localhost:3000/proxy/client/features',
+   urlHeaders: {
+     Authorization: 'bootstrap',
+   }
+  },
+});
+
+client.on('error', () => console.log("\x1b[31m", 'Unable to fetch feature toggles', "\x1b[0m"));
+client.on('warn', console.log);
+// client.on('changed', () => console.log('changed'));
```

Comment:
Should we trim out the commented code? Unless it's adding something useful to understanding what this is doing, I vote we remove it

Reply:
Yes, indeed!